pkg/downloader/downloader.go

```diff
@@ -357,37 +357,33 @@ func copyLocal(dst, src, ext string, decompress bool, description string, expect
 	if err != nil {
 		return err
 	}
-	if _, ok := Decompressor(ext); ok && decompress {
-		return decompressLocal(dstPath, srcPath, ext, description)
+	if decompress {
+		command := decompressor(ext)
+		if command != "" {
+			return decompressLocal(ctx, command, dstPath, srcPath, ext, description)
+		}
```

Comment:
I think an error should be returned if `command == ""`

Reply:
ping @alexandear

Reply:
Fixed

Reply:
Reverted this change because of https://github.com/lima-vm/lima/pull/2168#issuecomment-1939147257